<launch>
	<node pkg="spatiotemporalexploration" type="morse_scene_generator" name="scene_generator"  output="screen">
		<param name="positions_file" value="$(find SpatioTemporalExploration)/data/ww_objects.txt" />
		<param name="data_file" value="$(find SpatioTemporalExploration)/ww_july.txt" />
		<param name="initial_time" value="0" />
		<param name="time_step" value="1800" />
	</node>
	
</launch>
